frontend/src/modules/member/components/member-channels.vue

```diff
@@ -68,6 +68,15 @@
         :href="`https://reddit.com/u/${member.username.reddit}`"
         :as-link="true"
       />
+      <app-platform
+        v-if="!!member.username?.linkedin"
+        platform="linkedin"
+        track-event-name="LinkedIn"
```

Comment:
I think the `track-event-name` should be `Click member contact` and the `track-event-channel` should be `LinkedIn`. Could you just make sure which convention we are now using?

Reply:
Thanks for the suggestion, will fix!